### ISSUE TITLE
Key Binding Overwrites Important Default Mapping of Vim

### ISSUE THREAD

**Issue by Carltoffel** [rudrab/vimf90#15]
The last commit has broken the default mapping of vim. For example I no longer can use the 'ce' command (create executable gets triggered). I guess something went wrong with the mapping of the leader key.

**Comment by rudrab** [comment]
I will check that. Thanks for reporting.

**Comment by rudrab** [comment]
Oh, I thought I have misunderstood you. You can define the keybindings in your vimrc. For  Fortran90 menus, where `ce` is defined, you can overrite them as:
```
let fortran_compile="your value"
let fortran_exe=
let fortran_run=
let fortran_cla=
let fortran_dbg=
let fortran_make=
let fortran_makeProp=
let fortran_genProj=
```

**Comment by rudrab** [comment]
Please let me know if this solved your issue or revert back to me. 
Please close the issue and consider upvote the project if it solves your problem.

**Comment by Carltoffel** [comment]
This is not what i wanted to say. IMHO it is just not very smart for a plugin to overwrite a default vim command with something completely different. I would suggest to use the leader key of vim to initiate the commands of the plugin. Sure, I can rebind it, e.g. `let fortran_compile="<Leader>ce"`, but I think this should be the default for the plugin, and if a user wants to overwrite the default vim mapping, it is still possible.

**Comment by rudrab** [comment]
I agree it is not very good (or should be avoided by all means) to override default mappings. The keybinding  is actually defined in most generic way possible as:
`let b:fortran_compile  = get(g:,"fortran_compile", get(g:,'mapleader', '\')."cc")
`
so, if you define a `mapleader`, the problem is solved IMHO. Any suggestion is welcome though.

**Comment by Carltoffel** [comment]
I do have the `mapleader` defined (`let mapleader="\<SPACE>"`). Maybe this is the problem. If I set the `mapleader` to something else it works. Just a guess: Maybe your way of getting the `mapleader` is not aware of leader-keys longer than one character?

**Comment by Carltoffel** [comment]
Okay with `let mapleader=" "` I can fix my problem without changing the behaviour of vim. So maybe there is a problem with either the escape sequence or with the multi-character-leader.

Edit: I somehow was on the previous commit where it only worked with the `" "` leader. With the newest commit it only works if i set the `mapleader` to something like `m` but just a space does not work.

`let mapleader="\<SPACE>"` doesn't work
`let mapleader=" "` doesn't work
`let mapleader="m"` does work

**Comment by Carltoffel** [comment]
I don't know what your commands with `let b:fortran_compile = get(...` are doing exactly, but if I change them to be simply `let b:fortran_compile = "<Leader>cc"` etc. it works with all `mapleader`s mentioned above.

**Comment by mavens-group** [comment]
Resolved in commit 2a8c6a3. All key mappings are now strictly buffer-local (`<buffer>`) and mapped via standard `<Plug>(vimf90-...)` targets. They will no longer leak or overwrite default Vim normal-mode commands like `ce`.
